astro/src/content/docs/apis/login.mdx

```diff
@@ -1,6 +1,6 @@
 ---
-title: Login
-description: Learn about the APIs for authenticating users (logging them in) or tracking an SSO login (login ping) and multi-factor authentication.
+title: FusionAuth Login API Documentation
```

Comment:
```suggestion
title: Login
```

If you modify the name here, it modifies the name in the nav menu and looks weird. See image.

<img width="217" alt="Screenshot 2025-04-04 at 3 07 21 PM" src="https://github.com/user-attachments/assets/d75e20d8-e222-4ebf-92a7-8217493141b9" />